app/client/views/home/activities/sums/sums.js

```diff
@@ -1,59 +1,79 @@
 Template.homeResidentActivitySumsByType.created = function () {
-  // Get reference to template instance
-  var instance = this;
+    // Get reference to template instance
+    var instance = this;
+    // Get current home ID
+    instance.homeId = Router.current().params.homeId;
+    // set up home resident activity sums by type reactive variable
+    instance.homeResidentsActivitySumsByType = new ReactiveVar();
+    //instance.date = instance.data.date;
+    // Get home resident activity sums from server method
+    Meteor.call('getHomeResidentsActivitySumsByType', instance.homeId,instance.data.date.get(), function (error, activitySums) {
```

Comment:
This duplicate call to `getHomeResidentsActivitySumsByType` shouldn't be necessary, as `autorun` should run once in init.